server/src/logger/logger.js

```diff
@@ -0,0 +1,9 @@
+import log4js from 'log4js';
+
+const logger = (identifier) => {
+    const logger = log4js.getLogger(identifier);
+    logger.level = 'info';
```

Comment:
expose at `error` type too. Don't hardcode everything as info :)

Reply:
Thats logger level. Everything above INFO Level will be printed as logs.
https://www.npmjs.com/package/log4js#usage

Reply:
TRACE < DEBUG < INFO < WARN < ERROR < FATAL

Reply:
Store the 'info' variable in the env, and check whether its a dev or production or staging environment and show logs according to that.
Like in dev mode you might need the debug logs and traces.

Reply:
Okay!